packages/password/index.html

```diff
@@ -0,0 +1,31 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Password generator default settings</title>
+</head>
+<body>
+    <h1>Password generator default settings</h1>
+    <pre><code id="rules"></code></pre>
+    <pre><code id="parameters"></code></pre>
+    <pre><code id="password"></code></pre>
+    <script type="module">
+        import {generate} from "./index.js";
+        import {Password} from './lib/apple.password.js'
```

Comment:
This is the proof that with ESM, this module can be loaded into a browser environment without any need for a bundler - this makes testing/prototyping/demos much easier